Yotpo/Yotpo/view/adminhtml/templates/system/config/yotpobutton.phtml

```diff
@@ -0,0 +1,26 @@
+<?php
+?>
+
+<script type="text/javascript">
+    //<![CDATA[
+    function exportOrders() {debugger;
+    var params = {
+                form_key : FORM_KEY,
```

Comment:
It's a magento variable that is random and used for each session, you need to pass it when making an Ajax request